src/dmd/dsymbol.d

```diff
@@ -1347,7 +1382,7 @@ public:
     {
         //printf("ScopeDsymbol::syntaxCopy('%s')\n", toChars());
         ScopeDsymbol sds = s ? cast(ScopeDsymbol)s : new ScopeDsymbol(ident);
-        sds.comment = comment;
+        sds.addComment(comment);
```

Comment:
I don't think this is needed as you have the property setter in place.

Reply:
ok